lldb/test/API/functionalities/abbreviation/TestAbbreviations.py

```diff
@@ -80,7 +80,9 @@ def test_command_abbreviations_and_aliases(self):
         # Check a command that wants the raw input.
         command_interpreter.ResolveCommand(r"""sc print("\n\n\tHello!\n")""", result)
         self.assertTrue(result.Succeeded())
-        self.assertEqual(r"""script print("\n\n\tHello!\n")""", result.GetOutput())
```

Comment:
Since you are explicitly making the default lldb treat `script` as an alias for script execute, it might be good to test that as well.  We're already testing `sc` just above.